packages/shared/ReactComponentStackFrame.js

```diff
@@ -296,28 +290,12 @@ export function describeNativeComponentFrame(
   return syntheticFrame;
 }
 
-function describeComponentFrame(name: null | string) {
-  return '\n    in ' + (name || 'Unknown');
-}
-
 export function describeClassComponentFrame(ctor: Function): string {
-  if (enableComponentStackLocations) {
-    return describeNativeComponentFrame(ctor, true);
-  } else {
-    return describeFunctionComponentFrame(ctor);
-  }
+  return describeNativeComponentFrame(ctor, true);
 }
 
 export function describeFunctionComponentFrame(fn: Function): string {
```

Comment:
@sebmarkbage we need to keep this old export to support old devtools versions right?

Reply:
DevTools has its own copy in `DevToolsComponentStackFrame` so it can be deleted.

Reply:
This export seems used by other places still though.

Reply:
I missed this before merging somehow: https://github.com/facebook/react/pull/31781